settings.py

```diff
@@ -189,16 +189,32 @@
 # THis option is only for Adpod setup
 # Set ENABLE_DEAL_LINEITEM = True for creating deal line item
 # Defaults to False
-ENABLE_DEAL_LINEITEM = False
+ENABLE_DEAL_LINEITEM = True
```

Comment:
Should the default value be set to `False` as publisher might not want to enable deal setup always.

Reply:
Changed